sdk/Cargo.toml

```diff
@@ -74,24 +74,23 @@ dotenvy = { version = "0.15.7", default-features = false }
 fern-logger = { version = "0.5.0", default-features = false }
 
 [features]
-default = [ "std", "serde" ]
+default = [ "std", "serde", "tls", "client", "wallet" ]
```

Comment:
Do we keep std and serde here? It's already in the other features now

Reply:
Good point